### PR TITLE
[201911] Update Innovium device support

### PR DESCRIPTION
**Pull request by tonytitus**
**- Why I did it**
Add new device support for Innovium

**- How I did it**
Update Innovium platform and device dir

**- How to verify it**
Build and device bringup

**- Which release branch to backport (provide reason below if selected)**

- [ ] 201811
- [ ] 201911
- [ ] 202006

**- Description for the changelog**

Add new device delta_evs-a-32q56 support for Innovium
Update qos and buffer json files
Update Innovium config files

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by tonytitus** [comment]
retest vs please

**Comment by tonytitus** [comment]
retest vsimage please

File: device/x86_64-delta_evs-a-32q56-r0/led_proc_init.soc [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#ET_C032IF Port_Remap
+# Vlan set and port enable
+
+# led0 port order remap
+
+#ET_C032IF_LED
+
+#--------------------------------------------------------------------------------------------------
+#LED Auto link/up
+
```
Comment (jleveque) [inline device/x86_64-delta_evs-a-32q56-r0/led_proc_init.soc:10]: Everything is commented out. Is this intended? If so, why?

File: device/x86_64-delta_evs-a-32q56-r0/plugins/sfputil.py [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+        if ((port_num / 8) % 2 == 0):
+            os.popen("ipmitool raw 0x06 0x52 0x07 0x64 0x00 0x06 {0}".format(hex(reg_value)))
+        else:
+            os.popen("ipmitool raw 0x06 0x52 0x07 0x64 0x00 0x07 {0}".format(hex(reg_value)))
+
+        return True
+
+    def get_transceiver_change_event(self, timeout=0):
+
+        return False, {}
```
Comment (jleveque) [inline device/x86_64-delta_evs-a-32q56-r0/plugins/sfputil.py:196]: Why is this function not implemented?

**Review by jleveque — CHANGES_REQUESTED** [review]
Please fix LGTM alerts.

**Comment by tonytitus** [comment]
> Please fix LGTM alerts.

This is fixed

**Pull request merged**